y2m

```diff
@@ -138,7 +138,7 @@ do_clone() # {{{
   fi
   # clean shallow copy
   echo "Unshallowing git repo $moddir..."
-  cd $moddir && { git pull -q --unshallow; cd -; }
+  cd $moddir && { git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"; git pull -q --unshallow; cd -; }
```

Comment:
It is question if we want bigger initial clone or then slower pull. I think it is better here. I will decrease line size.